### ISSUE TITLE
Checking return of addTrajPoint and execute methods makes for ugly code

### ISSUE THREAD

**Issue by shaun-edwards** [shaun-edwards/moveit_simple#7]
Because `addTrajPoint` and `execute` return true on success, otherwise false, handling failures can make for ugly code: 

```
if( robot_.addTrajPoint(place_traj, offset, frame,         0.2, "dp_offset") &&
     robot_.addTrajPoint(place_traj, "wp_pick_approach",   0.4) &&
     robot_.addTrajPoint(place_traj, "wp_place_approach",  * 1.2) &&
     robot_.addTrajPoint(place_traj, "wp_place",          1.6))
  {
    if( robot_.execute(place_traj) )
       ...

  }
```

Perhaps we should consider changing to throwing exceptions.

**Comment by shaun-edwards** [comment]
@geoffreychiou, could you take a look at this.

**Comment by shaun-edwards** [comment]
@nsnitish, please take a look at this.  [Exceptions](https://www.tutorialspoint.com/cplusplus/cpp_exceptions_handling.htm) are and obvious solution to this.  I don't believe any of the standard exceptions apply, but you should probably inherit from `std::runtime_error`.  See [here](https://isocpp.org/wiki/faq/exceptions#how-exceptions) for some more background.

**Comment by shaun-edwards** [comment]
This has been addressed in PR #18
